keyboards/dark/magnum_ergo_1/magnum_ergo_1.c

```diff
@@ -0,0 +1,96 @@
+/* Copyright 2023 Gondolindrim <gondolindrim@acheronproject.com>
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#include "quantum.h"
+
+void board_init(void) {
+#ifndef LED_CAPS_LOCK_PIN
+    writePin(LED_CAPS_LOCK_PIN, 0);
+#endif
+
+#ifndef LED_INDICATOR_1
+    writePin(LED_INDICATOR_1, 0);
+#endif
+
+#ifndef LED_INDICATOR_2
+    writePin(LED_INDICATOR_2, 0);
+#endif
+
+#ifndef LED_INDICATOR_3
+    writePin(LED_INDICATOR_3, 0);
+#endif
+
+#ifndef LED_INDICATOR_4
+    writePin(LED_INDICATOR_4, 0);
+#endif
+
+#ifndef LED_INDICATOR_5
+    writePin(LED_INDICATOR_5, 0);
+#endif
+}
+
+bool led_update_kb(led_t led_state) {
+    bool res = led_update_user(led_state);
+    #ifdef LED_CAPS_LOCK_PIN
+    if(res) {
+        writePin(LED_CAPS_LOCK_PIN, led_state.caps_lock);
+    }
+    #endif
+    return res;
+}
```

Comment:
This isn't needed, with the define.
```suggestion
```

Reply:
Removed in [3be4f35](https://github.com/qmk/qmk_firmware/pull/22139/commits/3be4f35fe8420e4fad3760b1dd76525ea2687716)